linera-execution/src/lib.rs

```diff
@@ -865,18 +865,6 @@ impl Message {
         Message::System(message)
     }
```

Comment:
@andresilva91 It's kinda weird to have one of the two functions left.

Reply:
Sounds good! This PR adds it back: https://github.com/linera-io/linera-protocol/pull/1832